applications/GeoMechanicsApplication/custom_utilities/equation_of_motion_utilities.h

```diff
@@ -41,5 +41,13 @@ class GeoEquationOfMotionUtilities
                                          const Matrix& rMassMatrix,
                                          const Matrix& rStiffnessMatrix);
 
+    static Matrix CalculateStiffnessMatrixGPoint(const Matrix& rB,
+                                                 const Matrix& rConstitutiveMatrix,
+                                                 double        IntegrationCoefficient);
+
+    static Matrix CalculateStiffnessMatrix(const std::vector<Matrix>& b_matrices,
+                                           const std::vector<Matrix>& constitutive_matrices,
+                                           const std::vector<double>& integration_coefficients);
```

Comment:
To comply with the KratosStyle Guide, we need to rename the parameters as follows:
```suggestion
    static Matrix CalculateStiffnessMatrix(const std::vector<Matrix>& rBMatrices,
                                           const std::vector<Matrix>& rConstitutiveMatrices,
                                           const std::vector<double>& rIntegrationCoefficients);
```

Reply:
Opps.